aries-site/src/layouts/main/Layout.js

```diff
@@ -182,6 +195,9 @@ export const Layout = ({
                             topic={topic}
                             render={render}
                           />
+                          {status && wholeViewHistory[title].update &&
```

Comment:
```suggestion
                          {status && wholeViewHistory[title]?.update &&
```
Can we add a question mark here? This will prevent the app from crashing in the case where `wholeViewHistory[title].update` might not be defined.